test/unit/config/test__base.py

```diff
@@ -246,7 +246,7 @@ def test_wrap_args_with_ssh_agent_silent():
 def test_container_volume_mounting_with_Z(mock_isdir, mock_exists, mock_makedirs, tmpdir):
     rc = BaseConfig(private_data_dir=str(tmpdir))
     os.path.isdir = Mock()
-    rc.container_volume_mounts = ['project_path:project_path:Z']
+    rc.container_volume_mounts = ['project_path/:project_path/:Z']
```

Comment:
if we're implicitly handling the trailing slash, would it be worthwhile to test this without the trailing slash to make sure it handles it okay?

Reply:
tests added